tests/rptest/tests/full_disk_test.py

```diff
@@ -130,6 +138,67 @@ def test_write_rejection(self):
                 expect_blocked=True)
             self.full_disk.clear_low_space()
 
+    @cluster(num_nodes=3, log_allow_list=FDT_LOG_ALLOW_LIST)
+    def test_reset_health_monitor(self):
```

Comment:
Tagging @mmaslankaprv to take a look at this test as he was one of the last engineers to edit this file. However, anyone with context would also work as a reviewer.